src/openzaak/components/catalogi/management/commands/export.py

```diff
@@ -59,9 +60,7 @@ def handle(self, *args, **options):
                 _("The number of resources supplied does not match the number of IDs")
             )
 
-        factory = APIRequestFactory()
-        server_name = Site.objects.get_current().domain
-        request = factory.get("/", SERVER_NAME=server_name)
+        request = Request(build_fake_request())
```

Comment:
Sorry I don't understand why you wrap `build_fake_request()` in `Request` class. 
I thought it should already be a request and `Request` wasn't used in the old code

Reply:
(discussed in the office too)

it's because in this context DRF serializers are used, where typically the serializer context uses a DRF `Request`, which wraps around the Django request (see https://github.com/encode/django-rest-framework/blob/master/rest_framework/views.py#L385) and this serves to mimic that behaviour. `build_fake_request` returns a Django `HttpRequest` rather than a DRF `Request` as using the DRF Request outside of DRF context doesn't make much sense.